modules/phenyl-redux/src/local-state-updater.js

```diff
@@ -128,6 +130,34 @@ export class LocalStateUpdater<TM: TypeMap> {
     }
   }
 
+  /**
+   * Push network request promise.
+   */
+  static addUnreachedCommits<N: EntityNameOf<TM>>(state: LocalStateOf<TM>, commit: UnreachedCommit<N>): UpdateOperation {
+    const { entityName, id, commitCount } = commit
+    const enqueuedCount = state.unreachedCommits
+      .filter((c) => c.entityName === entityName && c.id === id)
+      .reduce((acc, c) => acc + c.commitCount, 0)
+    return {
+      $push: {
+        [createDocumentPath('unreachedCommits')]: {
+          entityName,
+          id,
+          commitCount: commitCount - enqueuedCount,
```

Comment:
What if `commitCount - enqueuedCount` becomes zero or negative value?

My idea is doing nothing when the `commitCount` would be zero or negative.